.github/buildEverything.py

```diff
@@ -92,6 +94,7 @@ def doBuild(makeDebug):
     ap.add_argument("-l", "--support-language", type=str, required=True, help="# model language to compile. (Enter 'all' to compile all language packs.)")
     ap.add_argument("-m", "--support-model", type=str, required=True, help="# model type to compile. (Enter 'all' to compile all model packs.)")
     ap.add_argument("-f", "--support-feature", type=str, required=False, default="", help="# feature to compile. (Enter a feature to compile.)")
+    ap.add_argument("-b", "--support-build", type=str, required=False, default="", help="# Build configuration to compile. (Enter a Debug/Release to compile.)")
```

Comment:
Not required, but no builds will be build without it. We should have a default or require it.

Reply:
If not given, build debugs and releases at the same time. This is the default I thought of.